mlflow/cli.py

```diff
@@ -584,7 +584,23 @@ def fetch_runs(token=None):
                 error_code=INVALID_PARAMETER_VALUE,
             )
         artifact_repo = get_artifact_repository(run.info.artifact_uri)
-        artifact_repo.delete_artifacts()
+        try:
+            artifact_repo.delete_artifacts()
+        except InvalidUrlException as iue:
+            click.echo(
+                click.style(
+                    f"The exception {iue} happen during deletion model artifact", fg="yellow"
```

Comment:
```suggestion
                    f"The exception {repr(iue)} happen during deletion model artifact", fg="yellow"
```
so that we can see the error class in log.

Reply:
Q: Shall we also bypass the exception when other type exception happens here ?

Reply:
I am confident that if the artifact URL is invalid, the artifact does not exist at the expected location. Thus, bypassing the InvalidURL exception during deletion is safe. However, I am concerned that bypassing other exceptions may lead to unexpected side effects.